ms-internal/containerized_webapi_image/README.md

```diff
@@ -0,0 +1,99 @@
+# Containerized CarbonAware WebApi
+
+The goal of this readme is to show how to build a container WebApi image that can be used to deploy the application into a container registry and that can be used later to run the service.
+
+## Build and List Runtime Image
+
+Use `docker` to build the WebApi images.
+(Note: Make sure the run `docker` at the root branch)
+
+```sh
+cd ./$(git rev-parse --show-cdup)/src
+docker build -t carbon_aware:v1 -f CarbonAware.WebApi/src/Dockerfile .
+```
+
+List `carbon_aware` image 
+
+```sh
+docker image ls carbon_aware
+REPOSITORY     TAG       IMAGE ID       CREATED             SIZE
+carbon_aware   v1        6293e2528bf2   About an hour ago   230MB
+```
+
+## Run WebApi Image
+
```

Comment:
Consider adding some information before 'For instance, ' Something like -
1. Run the image using 'docker run' command and provide the necessary configuration
For instance, .....